test/api_api_tokens_test.go

```diff
@@ -82,12 +65,16 @@ func Test_client_APITokensAPIService(t *testing.T) {
 
 		expireDate, err := time.Parse(time.RFC3339, "2021-01-01T00:00:00.000Z")
 		require.Nil(t, err)
-		tokenBody := client.CreateApiTokenSchemaOneOf{
-			Type:      "admin",
-			TokenName: "test admin token",
-			ExpiresAt: &expireDate,
+		environment := "development"
```

Comment:
This test was creating and deleting an admin token. Updated this to a frontend token so we can keep the test